server.js

```diff
@@ -4479,15 +4479,16 @@ cache(function(data, match, sendBadge, request) {
 
 // CircleCI build integration.
 // https://circleci.com/api/v1/project/BrightFlair/PHP.Gt?circle-token=0a5143728784b263d9f0238b8d595522689b3af2&limit=1&filter=completed
-camp.route(/^\/circleci\/(?:token\/(\w+))?[+\/]?project\/([^\/]+\/[^\/]+)(?:\/(.*))?\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/circleci\/(?:token\/(\w+))?[+\/]?project\/(github|bitbucket)\/([^\/]+\/[^\/]+)(?:\/(.*))?\.(svg|png|gif|jpg|json)$/,
 cache(function(data, match, sendBadge, request) {
-  var token = match[1]
-  var userRepo = match[2];  // eg, `BrightFlair/PHP.Gt`.
-  var branch = match[3];
-  var format = match[4];
+  var token = match[1];
+  var type = match[2]; // github OR bitbucket
```

Comment:
The addition of this piece of the URL will break existing links. Could you make that part optional, defaulting to GitHub, please?